src/App.jsx

```diff
@@ -14,33 +16,84 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_ALPHABETICALLY = 'name';
+const SORT_FIELD_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, sortField, isReversed) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField === SORT_FIELD_ALPHABETICALLY) {
+    preparedGoods.sort((a, b) => a.localeCompare(b));
+  } else if (sortField === SORT_FIELD_BY_LENGTH) {
+    preparedGoods.sort((a, b) => a.length - b.length);
+  }
```

Comment:
Consider will be better if you use switch case construction instead of if else if